doc/classes/CanvasItem.xml

```diff
@@ -334,7 +334,7 @@
 		<method name="get_global_mouse_position" qualifiers="const">
 			<return type="Vector2" />
 			<description>
-				Returns the global position of the mouse.
+				Returns the mouse's position relative to the origin of the [CanvasLayer] that this [CanvasItem] is in.
```

Comment:
This description is true but it might be not descriptive enough as:

>relative to the origin of the [CanvasLayer] that this [CanvasItem] is in

doesn't state in what way it's relative to the origin (origin is just a point). It may be not obvious that the returned position is described in the given `CanvasLayer`'s basis vectors. Meaning if that `CanvasLayer` is anyhow rotated, scaled, skewed, etc. (in other words when its basis is not standard) then there still might be some confusion about the result.

A description like:

>Returns the mouse's position in the coordinate space of the [CanvasLayer] that this [CanvasItem] is in.

doesn't have this issue (at least I think so) but it's probably not user-friendly as there are many no-math users (_"what is coordinate space?!"_). Maybe such description would be good if "coordinate space" would link to some page describing what it means? :thinking: 

My point is that current descriptions (the same issue with the other similar changes) are not fully descriptive. There's still missing information (relative in what way?) in them. I can't find wording to be both descriptive ("in the coordinate space") and user-friendly ("relative to") in a compact way. Maybe you could come up with something like that.

Reply:
@kleonc I've updated it to:
```Returns the mouse's position in the [CanvasLayer] that this [CanvasItem] is in using the coordinate system of the [CanvasLayer].```


>  Maybe such description would be good if "coordinate space" would link to some page describing what it means?

There is a tutorial page on [Matrices and transforms](https://docs.godotengine.org/en/stable/tutorials/math/matrices_and_transforms.html), but I'm not convinced it's helpful to link each method to this page.

Reply:
It's fine for me now.